app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/NavigationBarWidget.java

```diff
@@ -699,7 +699,10 @@ public void onFullScreen(GeckoSession session, boolean aFullScreen) {
             if (mIsResizing) {
                 exitResizeMode(false);
             }
-
+            int projection = VideoProjectionMenuWidget.getAutomaticProjection(SessionStore.get().getUriFromSession(session));
+            if (projection >= 0) {
+                enterVRVideo(projection);
```

Comment:
I like this - as opposed to automatically presenting when the page loads. smart thinking :+1: